README.md

```diff
@@ -487,7 +487,9 @@ Thirty-sixth Conference on Neural Information Processing Systems (NeurIPS), 2022
 - <a name="Messikommer23cvpr"></a>Messikommer, N., Fang, C., Gehrig, M., Scaramuzza, D.,  
 *[Data-driven Feature Tracking for Event Cameras](https://arxiv.org/abs/2211.12826)*,  
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2023. [PDF](https://rpg.ifi.uzh.ch/docs/CVPR23_Messikommer.pdf), [YouTube](https://youtu.be/dtkXvNXcWRY), [code](https://github.com/uzh-rpg/deep_ev_tracker).  
-
+- <a name="Pedersen23object"></a>Pedersen, J., Singhal, R., Conradt, J.,<br/>
```

Comment:
key value: Pedersen23nice

Reply:
This should be resolved with the latest commit :)